packages/cli/src/commands/pg/maintenance/index.ts

```diff
@@ -0,0 +1,29 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import {getAddon} from '../../../lib/pg/fetcher'
+import {essentialPlan} from '../../../lib/pg/util'
+import pgHost from '../../../lib/pg/host'
+import {MaintenanceApiResponse} from '../../../lib/pg/types'
+
+export default class Index extends Command {
+  static topic = 'pg';
+  static description = 'show current maintenance information';
+  static flags = {
+    app: flags.app({required: true}),
+  };
```

Comment:
Add missing `remote` flag and remove semicolon:

```suggestion
    remote: flags.remote(),
  }
```